snowpack/src/commands/dev.ts

```diff
@@ -718,6 +735,10 @@ export async function startServer(
       }
       return;
     } catch (err) {
+
+      logger.error(err.message);
```

Comment:
not needed? handleResponseError() should report on the error

Reply:
Correct, should have been removed, might not have pushed.